packages/at_client/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 3.0.75
+- feat: Replace encryption methods from EncryptionUtils with AtChops method
```

Comment:
3.0.75 should contain changes for enrollment. refer pub.dev

Reply:
Updated as part of the latest commit